package.json

```diff
@@ -50,7 +50,7 @@
     "pact:can-i-deploy": "node ./script/can-i-deploy.js",
     "pact:publish": "node ./script/publish-pacts.js",
     "pact:verify": "node ./script/verify-pacts.js",
-    "postinstall": "npm rebuild node-sass && node ./script/check-node-version.js",
+    "postinstall": "npm rebuild node-sass && node ./script/check-node-version.js && cd node_modules/web-components && npm install --only=prod && npm run-script build",
```

Comment:
This is because of how we are distributing `web-components`. Some more info here: https://github.com/department-of-veterans-affairs/component-library/pull/31#discussion_r584950386

Reply:
When can we expect that enhancement mentioned in the other thread?

Reply:
I'm not sure - it isn't planned at the moment. If this becomes a problem then I expect we will prioritize the enhancement for sooner rather than later. cc @cvalarida